portal-impl/src/com/liferay/portlet/messageboards/service/impl/MBMessageLocalServiceImpl.java

```diff
@@ -1979,11 +1979,13 @@ protected String getDiscussionMessageSubject(String subject, String body)
 
 		subject = HtmlUtil.extractText(body);
 
-		if (subject.length() <= 50) {
-			return subject;
+		int pos = Math.min(subject.length(), 50);
```

Comment:
Hi Rimi, I don't see why we need this Math.min comparison here. Could you please ellaborate on this?